Lesson31HomeWork/Lesson31HomeWork/ContentView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ContentView.swift
+//  Lesson31HomeWork
+//
+//  Created by Ernest Mihasenko on 26.06.22.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        studentsInfoTextFields
+    }
+    
+    @State var studentList = [Student]()
+    
+    @State var name = ""
+    @State var surname = ""
+    @State var grade = ""
+    
+    var studentsInfoTextFields: some View {
+        Group {
```

Comment:
It's better to put your elements in `VStack`. If you use just `Group`, you use the default `SwiftUI` and it can be changed in some iOS versions so it's better to be extra cautious and add `VStack` yourself